learning/tour-of-beam/frontend/lib/pages/tour/widgets/markdown/tob_markdown.dart

```diff
@@ -42,6 +43,11 @@ class TobMarkdown extends StatelessWidget {
       builders: {
         'code': MarkdownCodeBuilder(),
       },
+      onTapLink: (text, url, title) async {
+        if (url != null) {
+          await launchUrl(Uri.parse(url));
+        }
+      },
```

Comment:
Check how this works with internal links.
Also what URLs do we use in internal links? `/tour/java/module/group/unit`?